lib/mintacoin/accounts/crypto/stellar.ex

```diff
@@ -1,51 +1,80 @@
 defmodule Mintacoin.Accounts.Stellar do
   @moduledoc """
-  Mock implementation of the Stellar crypto functions for accounts
+  Implementation of the Stellar crypto functions for accounts
   """
 
-  alias Mintacoin.Accounts.{Crypto.AccountResponse, Keypair}
+  alias Mintacoin.Accounts.Crypto.AccountResponse
+  alias Stellar.{Horizon, Horizon.Transaction, KeyPair, TxBuild}
+
+  @type status :: :ok | :error
+  @type stellar_response :: map()
+  @type publi() :: map()
+  @type public_key :: String.t()
+  @type secret_key :: String.t()
+  @type error :: {:error, any()}
 
   @behaviour Mintacoin.Accounts.Crypto.Spec
 
   @impl true
   def create_account(_opts) do
-    {:ok, {secret_key, public_key}} = Keypair.random()
+    {fund_public_key, fund_private_key} = fund_key_pair_from_system()
+    {public_key, secret_key} = KeyPair.random()
+
+    source_account = TxBuild.Account.new(fund_public_key)
+    {:ok, seq_num} = Horizon.Accounts.fetch_next_sequence_number(fund_public_key)
+    sequence_number = TxBuild.SequenceNumber.new(seq_num)
+
+    operation =
+      TxBuild.CreateAccount.new(
+        destination: public_key,
+        starting_balance: 1
+      )
+
+    signature =
+      {fund_public_key, fund_private_key}
+      |> TxBuild.Signature.new()
```

Comment:
@jhonatan-kmt I this looks better:
```suggestion
      TxBuild.Signature.new(fund_keypair)
```